tests/test_fields.py

```diff
@@ -171,13 +172,28 @@ def test_model_with_image_and_format(self):
     def test_model_rgb_formats(self):
         obj = ColorFieldRGBFormat(
             color_rgb="rgb(123, 123, 123)",
-            color_rgba="rgba(5, 10, 128, 0.5)",
+            color_rgba="rgba(128, 199, 255, 0.55)",
```

Comment:
Not sure why this passes, I would have expected that we need to increase the default `max_length`, as it's 25 chars long:

https://github.com/fabiocaccamo/django-colorfield/blob/a019212bcded27949b64e4fe280c3d4cf97dd25c/colorfield/fields.py#L44

Reply:
Right... SQLite does not enforce the length of a VARCHAR

Reply:
At this point the field `max_length` value must be updated to 25.